katana/data/nomads_hrrr.py

```diff
@@ -0,0 +1,92 @@
+import logging
+import os
+from copy import deepcopy
+from datetime import datetime
+
+from katana import utils
+from katana.data.data_base import BaseData
+from katana.grib_crop_wgrib2 import create_new_grib
+from katana.wind_ninja import WindNinja
+
+
+class NomadsHRRR(BaseData):
+
+    def __init__(self, config, topo):
+        """Init the NomadsHRRR class
+
+        Arguments:
+            config {dict} -- dictionary of configuration options
+            topo {Topo class} -- katana.data.topo.Topo class instance
+        """
+
+        self._logger = logging.getLogger(__name__)
+
+        self.config = config
+        self.topo = topo
+
+        self.buffer = self.config['input']['hrrr_buffer']
+        self.directory = self.config['input']['hrrr_directory']
+        self.make_new_gribs = self.config['output']['make_new_gribs']
+        self.nthreads_w = self.config['input']['hrrr_num_wgrib_threads']
+
+        self.start_date = self.config['time']['start_date']
+        self.end_date = self.config['time']['end_date']
+
+        # create an hourly time step between the start date and end date
+        self.date_list = utils.daterange(self.start_date, self.end_date)
+
+        # create a daily list between the start and end date
+        self.day_list = utils.daylist(self.start_date, self.end_date)
+
+        self.out_dir = self.config['output']['out_location']
```

Comment:
In truly inheritance fashion, this should move to the `BaseData` object.
Then this initializer can call this via `super()`.

Reply:
Overall a lot of this and the `WRFout` class can be combined and have it in the `__init__` call. Let me know if you want help on this.

Reply:
I thought about doing that then decided against it, but you're forcing my hand!

Reply:
Moved similar conifg options to `BaseData`